src/docs/docs/getting-started/vscode-ssh.md

```diff
@@ -73,47 +83,67 @@ The following will show you how to create and connect to a virtual machine (VM)
     - Port Range: `8443`
     - Source: `My IP`
 
-  ![](https://seneca-cdot-telescope.s3.amazonaws.com/vscode-ssh/2021-10-26+12_54_12-vscode-ssh.md+-+telescope+-+Visual+Studio+Code.png)
+  ![AWS Security Groups](https://seneca-cdot-telescope.s3.amazonaws.com/vscode-ssh/2021-10-26+12_54_12-vscode-ssh.md+-+telescope+-+Visual+Studio+Code.png)
```

Comment:
Are we going to run into issues with people who don't have a static IP and how we're telling them to do their security group (i.e., using My IP if it changes between sessions)?

Reply:
I put a note stating that if their IP address changes, they should update their Security Group. Common causes of IP address changes are:
- changing internet providers
- route rebooting (either manually or power outage)

I want to somehow teach everyone about security groups and IP addresses because they are often the culprits when something goes wrong...